src/schema-conversion/convertResource.spec.ts

```diff
@@ -216,9 +220,13 @@ describe('convertValue', () => {
 		['100', 'integer', 100],
 		['1970-01-01T00:00:00Z', 'timestamp', 0],
 		['', '', null],
-	])(
-		"should convert value: '%s' which is type: '%s' to: %s",
-		(value, type: any, expected) =>
-			expect(convertValue(value, type)).toEqual(expected),
-	)
+	] as [
+		string,
+		'integer' | 'number' | 'boolean' | 'string' | 'timestamp',
+		unknown,
+	][]) {
+		void it(`should convert value '${value}' which is type '${type}' to '${expected}'`, () => {
+			assert.deepEqual(convertValue(value, type), expected)
+		})
```

Comment:
```suggestion
		void it(`should convert value '${value}' which is type '${type}' to '${expected}'`, () => assert.equal(convertValue(value, type), expected))
```

No deep equality needed.